### PR TITLE
Add kinesis source 3.0 docs

### PR DESCRIPTION
**Pull request by shibd**
Google docs: https://docs.google.com/document/d/1jXQelw9q5TWGkolCQ5Et-XJqTmDh9fYxju64Us7MBP8/edit#heading=h.sb8cpui9n7ur

### CODE REVIEW

File: connectors/kinesis-source/3.0/kinesis-source.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    ByteBuffer data = ByteBuffer.wrap("test-kinesis-data".getBytes("UTF-8"));
+    // doesn't block       
+    kinesis.addUserRecord("Your kinesis stream name", "myPartitionKey", data);
+  }
+  kinesis.flush();
+  Thread.sleep(60000);
+}
+```
+
+### 3. Show data by Pulsar client
```
Comment (momo-jun) [inline connectors/kinesis-source/3.0/kinesis-source.md:99]:
```suggestion
### 3. Show data using Pulsar client
```

File: connectors/kinesis-source/3.0/kinesis-source.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+The Kinesis source connector pulls data from Amazon Kinesis and persists data into Pulsar. For more information about connectors, see [Connector Overview](https://docs.streamnative.io/docs/connector-overview).
+
+![](/images/connectors/kinesis-source.png)
+
+This connector uses the [Kinesis Consumer Library](https://github.com/awslabs/amazon-kinesis-client) (KCL) to consume messages. The KCL uses [DynamoDB](https://docs.aws.amazon.com/streams/latest/dev/shared-throughput-kcl-consumers.html) to track checkpoints for consumers,
+and uses [CloudWatch](https://docs.aws.amazon.com/streams/latest/dev/monitoring-with-cloudwatch.html) to track metrics for consumers.
+
+This document introduces how to get started with creating an AWS Kinesis source connector and get it up and running.
+
+> #### Note
```
Comment (momo-jun) [inline connectors/kinesis-source/3.0/kinesis-source.md:34]: Does it make sense to align the style/format for note? Now we have three different ones on a single page. See [2](https://github.com/streamnative/pulsar-hub/pull/296/files#diff-9888b895b61b8758a66b48d2db388cf2ac4ad456216bf8ebb12d06b14e801f3fR70) and [3](https://github.com/streamnative/pulsar-hub/pull/296/files#diff-9888b895b61b8758a66b48d2db388cf2ac4ad456216bf8ebb12d06b14e801f3fR101).

File: images/connectors/kinesis-source.png
Comment (momo-jun) [inline images/connectors/kinesis-source.png:1]: The text and icon pairs don't match.

File: connectors/kinesis-source/3.0/kinesis-source.md (hunk 2 above)
Comment (shibd) [reply connectors/kinesis-source/3.0/kinesis-source.md:34]: Yes, Thanks for your feedback.

**Review by momo-jun — APPROVED** [review]

**Pull request merged**